src/engine_uring.cpp

```diff
@@ -89,6 +89,11 @@ struct connection_t;
 struct engine_t;
 struct automata_t;
 
+static constexpr std::size_t uring_acpt_tag_k{0};
+static constexpr std::size_t uring_recv_tag_k{1};
+static constexpr std::size_t uring_send_tag_k{2};
+static constexpr std::size_t uring_stat_tag_k{3};
```

Comment:
Maybe better to use an `enum class using_verb_t : std::uintptr_t {}`, and later reuse that enum-type instead of `uint64_t type`  in `stage_t` and `mutex_t`. Would make things more readable.